code/model/modelread.cpp

```diff
@@ -3403,7 +3403,7 @@ int model_load(const  char* filename, int n_subsystems, model_subsystem* subsyst
 					dl2--;	// Start from 1 up...
 					if (dl2 >= sm1->num_details ) sm1->num_details = dl2+1;
 					sm1->details[dl2] = j;
-  				    mprintf(( "Submodel '%s' is detail level %d of '%s'\n", sm2->name, dl2 + 1, sm1->name ));
+  				    nprintf(("Model", "Submodel '%s' is detail level %d of '%s'\n", sm2->name, dl2 + 1, sm1->name ));
```

Comment:
Not sure how I feel about this one being hidden.
Accidental LOD-ing is _the_ most common cause for submodels misbehaving I have seen, and it being hidden makes it a lot harder to spot on a glance from logs from non-tech-savvy people.

Reply:
Good point

Reply:
Alright. I think the message is awkwardly formatted and confusing in isolation, but that's a seperate issue. I will revert.